Lib/uuid.py

```diff
@@ -728,10 +728,10 @@ def uuid1(node=None, clock_seq=None):
 
     global _last_timestamp
     import time
-    nanoseconds = int(time.time() * 1e9)
+    nanoseconds = time.time_ns()
```

Comment:
Why `int(time.time() * 1e9) == time.time_ns()` is `False`?  Must be `True`, isn't?

Reply:
ho! you are right, I was confused